CHANGELOG.md

```diff
@@ -221,7 +223,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Initial creation.
 
-[Unreleased]: https://github.com/xmidt-org/tr1d1um/compare/v0.9.0...HEAD
+[v0.9.1]: https://github.com/xmidt-org/tr1d1um/compare/v0.9.0...v0.9.1
```

Comment:
You'll still want to included the `unreleased` line above this one. It will look like this
`[Unreleased]: https://github.com/xmidt-org/tr1d1um/compare/v0.9.1...HEAD`